parser/charset/encoding_base.go

```diff
@@ -34,6 +34,10 @@ type encodingBase struct {
 	self Encoding
 }
 
+func (b encodingBase) MbLen(_ string) int {
+	return 0
+}
+
```

Comment:
```suggestion
```
It is better not to implement in `encodingBase` because the parent may forget this.

Reply:
I think this is what I need, it is the default implement